.github/workflows/on-push-publish-to-npm.yml

```diff
@@ -0,0 +1,19 @@
+name: on-push-publish-to-npm
+on:
+  push:
+    branches:
+      - master
```

Comment:
This will cause the merge of this PR (and subsequent PRs) to deploy to npm, automatically, right? Am I free to create `develop` branches and prescribe a release process/workflow? I don't mind keeping `master` as _the_ branch that is always evergreen but I can imagine we don't want all PRs to automatically get released, right?

Reply:
This workflow will run anytime there is a commit to the `master` branch that affects the `package.json` file. Lines 6-7 of the workflow. If the publish action (line 17) detects the version of your package has changed it will publish to npm for you.

Reply:
Oh ok I see. So I can update dependencies or make other changes to `package.json` so long as I don't modify the version. Once the version is bumped, it will run a release?